base/deprecated.jl

```diff
@@ -1355,6 +1362,15 @@ module DSP
 end
 using .DSP
 export conv, conv2, deconv, filt, filt!, xcorr
+@doc """
+DFT will be removed from Base. Use the package with the same name instead.
```

Comment:
There's no package called DFT. The functionality has been moved to AbstractFFTs.